bin/cli.js

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
+  [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '--reporter <value>',        'Reporter name'],
+  [ '--log-level <value>',       'Log level'],
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error':   [ 'LOG_ERROR', 'errors-only' ],
+  'warn':    [ 'LOG_WARN', 'minimal' ],
+  'info':    [ 'LOG_INFO', 'normal' ],
+  'debug':   [ 'LOG_DEBUG', 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
+
+const isLogAtLeast = (level1, level2) => {
+  if (logLevels.indexOf(level1) === -1) level1 = 'disable';
+  if (logLevels.indexOf(level2) === -1) level2 = 'disable';
+  return logLevels.indexOf(level1) >= logLevels.indexOf(level2);
+};
+
+const merge = (arr1 = [], arr2 = []) => common.merge(arr1, arr2);
```

Comment:
Why don't you use `common.merge` itself?

Reply:
Because config file or command-line options may not contain required parameters and thus,  resulting in `undefined` which will lead to `TypeError` in `common.merge`. This is because the number of arguments will still count `undefined` as one of the arguments. e.g.
```js
common.merge([1,2,3], []);
```
will work and
```js
common.merge([1,2,3], undefined);
```
won't. And as for me, having default arguments in a separate function is better than in every `common.merge` call
```js
common.merge(arr1 || [], arr2 || []);
```
Another option is to change `common.merge` itself
```js
const merge = (...args) => {
  const unique = new Set();
  const ilen = args.length;
  for (let i = 0; i < ilen; i++) {
    const arr = args[i];
    if (!Array.isArray(arr)) continue;
    for (let j = 0; j < arr.length; j++) {
      unique.add(arr[j]);
    }
  }
  return [...unique];
};
```